### DISCUSSION TITLE
Add SubWallet to the Custom Wallet page on RainbowKit docs

### DISCUSSION THREAD

**Discussion by djhatrang** [rainbow-me/rainbowkit#1771] [Feedback]
Hi team, 

I'm creating this issue to add SubWallet to the Custom Wallet List on RainbowKit docs: https://www.rainbowkit.com/docs/custom-wallet-list

Some information about SubWallet: 
- SubWallet was integrated to RainbowKit in October 2023. 
- SubWallet is the leading comprehensive non-custodial wallet solution for Polkadot, Substrate & Ethereum ecosystems with 250K+ active users across 3 platforms: mobile app, browser extension and web dashboard. Users can easily sync their accounts across platforms.
- SubWallet now supports 150+ networks (both EVM and Polkadot SDK-based networks) and 400+ tokens. 70+ dApps have integrated SubWallet.
- More information can be found on our website: https://www.subwallet.app/

Looking forward to seeing SubWallet added to the Custom Wallet List on RainbowKit docs!

**Comment by 0xmkh** [comment]
@djhatrang Thanks for pointing this out. We don't show it on the custom wallet list, but we'll make sure to update it 🙏

**Reply by djhatrang** [reply]
Thank you! Let me know once it's updated so we can inform our partners :")

**Reply by 0xmkh** [reply]
Yeah will do
